_scalingo_api/cron_tasks.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Cron tasks
```

Comment:
Title in English have first letters of all words in upper case
```suggestion
title: Cron Tasks
```

https://www.notion.so/scalingooriginal/Writing-Documentation-at-Scalingo-bb7985d6ea16455894417e36ede09cea